src/pypn_habref_api/models.py

```diff
@@ -109,7 +109,7 @@ class BibListHabitat(DB.Model):
     __table_args__ = {"schema": "ref_habitats"}
     id_list = DB.Column(DB.Integer, primary_key=True)
     list_name = DB.Column(DB.Unicode)
-    habitats = DB.relationship("Habref", secondary=cor_list_habitat)
+    habitats = DB.relationship("Habref", secondary=cor_list_habitat, overlaps="lists")
```

Comment:
```suggestion
    habitats = DB.relationship("Habref", secondary=cor_list_habitat, back_populates="lists")
```

Rajouter également `back_populates="habitats"` sur la relationship `Habref.lists`.